python/cudf/cudf/core/column/string.py

```diff
@@ -11,39 +11,37 @@
 
 import nvstrings
 
-import cudf._libxx as libcudfxx
-import cudf._libxx.string_casting as str_cast
-from cudf._libxx.nvtext.generate_ngrams import (
+import cudf._lib as libcudf
+import cudf._lib.string_casting as str_cast
+from cudf._lib.nvtext.generate_ngrams import (
     generate_ngrams as cpp_generate_ngrams,
 )
-from cudf._libxx.nvtext.ngrams_tokenize import (
+from cudf._lib.nvtext.ngrams_tokenize import (
```

Comment:
I suppose the instances of `cudf._lib` could be replaced with `libcudf` here for consistency.

Reply:
Yes -- but we don't follow that pattern anywhere else so it would stick out if we did in just `string.py`

Reply:
Actually, I noticed it because other files (`numerical.py`, `datetime.py`, `column.py`, etc) in the same directory did it the way I suggested, so I meant to be consistent with that. This was only a minor suggestion, so feel free to ignore. :smile: